rust/common-runtime/src/ifc/value.rs

```diff
@@ -0,0 +1,109 @@
+use crate::CommonRuntimeError;
+use common_protos::common as proto;
+
+/// An intrinsic value type within a Common Runtime
+#[derive(PartialEq, Clone, Debug)]
+pub enum Value {
```

Comment:
This seems like an unnecessary conflation of what is essentially an implementation detail of the `common:*` environment API and IFC constructs. `Data` is the IFC counterpart that wraps up something like a `Value`, right? `Value` is a host-side implementation of something that is made available within the Wasm runtime (in other words, orthogonal to IFC policy evaluation / enforcement).